troubleshooting-tap/troubleshoot-using-tap.hbs.md

```diff
@@ -594,3 +594,81 @@ take "/" at the end of the string.
 ### Solution
 
 Do not append "/" to the end of the string.
+
+---
+
+## <a id='cluster-issuer-trust-issues'></a> x509: certificate signed by unknown authority
+
+### Explanation
+
+In TAP 1.4, a [Shared Ingress Issuer](../release-notes.hbs.md#1-4-0-tap-new-features) was inttroduced for securing ingress communication by default. The Certificate Authority for it is generated as self-signed and thus some procedures in TAP will result in errors such as:
+* `connection refused`
+* `x509: certificate signed by unknown authority`
+
+### Solution
+
+There are two procedures that can mitigate the issue.
+
+* Configure the Shared Ingress Issuer's Certificate Authority as a trusted Certificate Authority
+  * This is the recommended option for a secure instance
+* Disable the shared ingress issuer
+  * This option is fast and convenient but is not recommended. This option is documented for testing purposes only.
+
+#### Configure the Shared Ingress Issuer's Certificate Authority as a trusted Certificate Authority
+
+The following procedure results in the Shared Ingress Issuer's Certificate Authority to be trusted within TAP.
+
+1. Extract the ClusterIssuer's Certificate Authority from cert-manager
+
+  ```console
+  kubectl get secret tap-ingress-selfsigned-root-ca -n cert-manager -o yaml | yq .data | cut -d' ' -f2 | head -1 | base64 -d
+  ```
+
+1. Add the certificate to the list of trusted certificate authorities. This is done by updating your tap-values.yml to append the above certificate authority to the field `shared.ca_cert_data`
+
+1. Pause the meta package's reconciliation. This prevents Tanzu Application Platform from reverting to the original values.
```

Comment:
I don't think you want to turn off the reconciliation.  You have to if you are trying to reinstall a specific component and don't want tap to reset it back to the original values,  but in this case you want the meta package to pickup that change and propagate it down to all the packages.